theseus_gui/src/components/ui/InstanceCreationModal.vue

```diff
@@ -0,0 +1,205 @@
+<template>
+  <Modal ref="modal" header="Create Instance">
+    <div v-if="showContent" class="modal-body">
+      <div class="image-upload">
+        <Avatar :src="display_icon" size="md" :rounded="true" />
+        <div class="image-input">
+          <Button @click="upload_icon()">
+            <UploadIcon />
+            Upload Icon
+          </Button>
+          <Button @click="reset_icon">
+            <XIcon />
+            Remove Icon
+          </Button>
+        </div>
+      </div>
+      <div class="input-row">
+        <p class="input-label">Game Version</p>
+        <DropdownSelect v-model="game_version" :options="game_versions" />
+      </div>
```

Comment:
For this I would use a multiselect dropdown as it has search and scrolling. Right now the dropdown glitches off the screen